plasma_framework/test/src/framework/BlockController.test.js

```diff
@@ -115,4 +126,17 @@ contract('BlockController', ([_, other]) => {
             );
         });
     });
+
+    describe('new vault', () => {
```

Comment:
minor: the tests in this file is structured by function name and I think it would be nice if we put this under `registerVault` so the result of test can easily serve as a document of the function here. I think structuring test in this style might help audit process. But I will leave this to you as that is more or less just my personal preference.